modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -1,10 +1,10 @@
 = Debugging kernel crashes using kdump
 
-. Memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
+. Memory must be reserved for the crash kernel during booting of the first kernel. `crashkernel=auto` is likely insufficient memory on Fedora CoreOS. It is recommended to start testing with 300M. Kernel arguments can be provided like this:
 +
 [source, bash]
 ----
-sudo rpm-ostree kargs --append='crashkernel=256M'
+sudo rpm-ostree kargs --append='crashkernel=512M'
```

Comment:
Should we just use 300M here to make it easy to copy/paste if that's what we recommend?